### PR TITLE
Add APIs to load HTML and URL

### PR DESCRIPTION
**Pull request by tkf**
To implement "single window mode" for ElectronDisplay.jl (as discussed in https://github.com/fredo-dedup/VegaLite.jl/issues/141), I think we need an API to load HTML in an existing Window.  This PR adds `loadhtml` to do that and also `loadurl` which is used by `loadhtml`.  It's first time I touch Electron API and I don't know much about Javascript.  I hope this is a reasonable way to do it.

### CODE REVIEW

**Comment by davidanthoff** [comment]
Yes, I think that makes sense, thanks a lot!

In terms of API, I think we should have only one ``load`` function, and when you pass a ``URI`` it loads that, and if you pass an ``AbstractString`` it would essentially do your current ``loadhtml``. That would match the existing constructor calls. And once I've merged https://github.com/davidanthoff/Electron.jl/pull/18, one can also pass paths by type.

**Comment by tkf** [comment]
Ah yes, that is a cleaner API.  I updated the patch.

File: src/Electron.jl [hunk 1]
```diff
@@ -257,6 +257,26 @@ function Base.run(win::Window, code::AbstractString)
     return retval["data"]
 end
 
+"""
+    load(win::Window, url::AbstractString)
```
[the remark this answers is not in the document]
Comment (tkf) [reply src/Electron.jl]: How did I possibly miss that!

File: src/Electron.jl [hunk 2]
```diff
@@ -257,6 +257,26 @@ function Base.run(win::Window, code::AbstractString)
     return retval["data"]
 end
 
+"""
+    load(win::Window, uri::AbstractString)
```
Comment (davidanthoff) [inline src/Electron.jl]: I think this is still wrong, right?

Comment (tkf) [reply src/Electron.jl]: Sorry, I was sloppy and miss read that you are talking about function parameter.

**Pull request merged**